migrations/0058_order_history.up.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS order_history (
+    id serial PRIMARY KEY,
```

Comment:
`serial` fields [are implicitly](https://www.postgresql.org/docs/14/datatype-numeric.html#DATATYPE-SERIAL) `NOT NULL`